Sources/SpeziHealthCharts/MetricChart/Models/HealthChartStyle.swift

```diff
@@ -0,0 +1,23 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+
+
+public struct HealthChartStyle: Sendable {
```

Comment:
Eager to see how the health chart style turn out. What's the general idea, just having some parameters that control how the chart is rendered?